crates/cibse_guide_e/src/chapter_6/appendix/a_simple_case.rs

```diff
@@ -0,0 +1,60 @@
+pub fn floor_area(w1: f64, w2: f64) -> f64 {
+    return w1 * w2
+}
+
+pub fn equation_floor_area(a_f: String, w1: String, w2: String) -> String {
```

Comment:
```suggestion
pub fn floor_area_equation(a_f: String, w1: String, w2: String) -> String {
```
I just feel this reads better